examples/gosherve-demo/slides/gosherve-demo-slides.md

```diff
@@ -0,0 +1,908 @@
+# Charmed Operator Development Workshop

```

Comment:
I wonder if we want to have the copy of the presentation in the monorepo.
Maybe we could reference the slides instead?